### PR TITLE
Fixes #17290 - Usability improvements

### PR DESCRIPTION
**Pull request by ares**
I started with some UI changes some time ago, since we now have dedicated person for it and I fail to find time for doing more I decided to send current version. Feel free to take what you find useful.
tasks_details

notable changes:
- owner is a link to user, reworded to triggered by
- result is only displayed if the task state is stopped, "?" otherwise, result has icon
- dates are printed more user friendly, grouped in right column
- id and label are moved to Raw tab
- progress bar changes according to patternfly
- sub tasks, parent task links converted to buttons (next to cancel)

### CODE REVIEW

File: app/views/foreman_tasks/tasks/_details.html.erb [hunk 1]
```diff
@@ -79,72 +85,65 @@
...
+  <div class="col-md-6">
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Name") %>:</span>
+      <span class="param-value"><%= format_task_input(@task, true) %></span>
+    </div>
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Result") %>:</span>
+      <span class="param-value">
+        <% if @task.state != 'stopped' %>
+          <%= content_tag(:i, '&nbsp'.html_safe, :class => 'task-status pficon-help') %>
```
Comment (adamruzicka) [inline app/views/foreman_tasks/tasks/_details.html.erb]: Is there a way how a user could know a task got paused? If I'm reading this right having a paused task would just show a pretty question mark in the result and make the resume button clickable.

File: app/views/foreman_tasks/tasks/_details.html.erb [hunk 2]
```diff
@@ -79,72 +85,65 @@
...
+      <span class="param-value"><%= time_in_words_span @task.started_at.try(:in_time_zone) %></span>
+    </div>
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Ended at") %>:</span>
+      <span class="param-value"><%= time_in_words_span @task.ended_at.try(:in_time_zone) %></span>
+    </div>
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Start before") %>:</span>
+      <span class="param-value"><%= @task.start_before.nil? ? '-' : time_in_words_span(@task.start_before) %></span>
+    </div>
```
Comment (adamruzicka) [inline app/views/foreman_tasks/tasks/_details.html.erb]: How about adding another field showing how long the execution took?

**Comment by adamruzicka** [comment]
The task name will need to be truncated

long

**Comment by ares** [comment]
Thanks @adamruzicka, name is being truncated, I moved duration to Raw tab and highlighted headers there too. I added empty line between progress bar and attributes. Please take another look.

**Comment by ares** [comment]
I updated the label title to display the full name instead of raw label (that is to be found under raw tab). I also updated the locks page, see attached screenshot
locks

**Comment by adamruzicka** [comment]
ACK from me, let's wait for Mr. Jenkins' approval and get it in

**Comment by ares** [comment]
@adamruzicka I fixed the show vs 2pane styling. It's extracted to new css file which is included on both pages.

**Comment by adamruzicka** [comment]
Great, let's get this in

**Comment by iNecas** [comment]
Seems the issue with tests failures was temporary, the re-run went green http://ci.theforeman.org/job/test_plugin_pull_request/3060/

**Comment by ares** [comment]
:champagne:

**Pull request merged**